data/api/client-server/room_initial_sync.yaml

```
@@ -7,10 +7,12 @@ paths:
     get:
       summary: Snapshot the current state of a room and its most recent messages.
       description: |-
+        This api should now exclusively be used for "peeking" into viewable rooms you're not a member of ([Room Previews](/client-server-api/#room-previews))
+
         Get a copy of the current state and the most recent messages in a room.
 
```

I think we still need to start with a description of what the endpoint actually does.

```suggestion
        Get a copy of the current state and the most recent messages in a room.

        This endpoint should now exclusively be used for "peeking" into viewable rooms
        you're not a member of ([Room Previews](/client-server-api/#room-previews)).
```